public/javascript/idea.js

```diff
@@ -0,0 +1,31 @@
+class Idea {
```

Comment:
ES6 & Vanilla JS...Keeping jQuery in for the moment as fingers crossed we can use it soon :D